packages/components/src/truncate/test/__snapshots__/truncate.js.snap

```diff
@@ -0,0 +1,192 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`props should render correctly 1`] = `
```

Comment:
Not that important but for me tests that dump CSS are not great tests, this very fragile and it's testing implementation details that can change often.

Reply:
I personally don't like how big this snapshot output becomes. There are two possible solutions:
- we use snapshot testing as little as possible and assert for the subset of produced output
- we create a custom snapshot serializer that removes the part that obscures the output

It might also help to use inline snapshots if we make them more readable a few lines long.

Reply:
These tests are actually useless anyway as the snapshots are all the same (though most certainly not meant to be) I’ve been spending time refactoring them to use assertions rather than snapshots. I personally think snapshots are not useful most of the time as people add and change them indiscriminately.